app/models/collection_item.rb

```diff
@@ -188,7 +188,7 @@ def notify_of_status_change
         notify_of_reveal
       end
     end
-    if anonymous_changed?
```

Comment:
If you take out "anonymous_changed?" then this will send out emails every time the collection item is saved (whether it was ever anonymous in the first place or not), because it's a simple after_update hook. That's probably not what you want.